### PR TITLE
Proposed 0.81.0

### PR DESCRIPTION
**Pull request by nbougalis**

### CODE REVIEW

**Comment by mtrippled** [comment]
:+1:

**Comment by scottschurr** [comment]
This is really picky, but in RELEASENOTES.md, the link underneath `[compile from source]` leads here: https://wiki.ripple.com/Rippled_build_instructions.  That link says in a big red field at the top, "This web site is no longer actively supported...".  Do we have a better link for rippled build instructions?

File: RELEASENOTES.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ If you are using Red Hat Enterprise Linux 7 or CentOS 7, you can [update using `
...
+The `rippled` 0.81.0 release introduces changes that improve the scalability of the XRP Ledger and transitions the recommended validator configuration to a new hosted site, as described in Ripple's [Decentralization Strategy Update](https://ripple.com/dev-blog/decentralization-strategy-update/) post.
+
+**New and Updated Features**
+
+- New hosted validator configuration.
+
+
+**Bug Fixes**
+
+- Filter SQL results in lieu of a search clause ([#2312](https://github.com/ripple/rippled/isses/2312))
```
Comment (scottschurr) [inline RELEASENOTES.md]: The URL leads to a 404.  Is "isses" really supposed to be part of the path?

Comment (ximinez) [reply RELEASENOTES.md]: The description isn't accurate anymore either. The updated commit message is better: `Optimize queries for account_tx to work around SQLite query planner`. Or simplified as `Optimize long-running SQL queries`

**Comment by nbougalis** [comment]
Thanks for the comments. Will update

**Comment by MarkusTeufelberger** [comment]
> Do we have a better link for rippled build instructions?

Unfortunately not at the moment.

**Comment by MarkusTeufelberger** [comment]
The data format at https://vl.ripple.com is not documented anywhere I could find on https://ripple.com/build/ @mDuo13

File: RELEASENOTES.md (hunk 1 above)
Comment (MarkusTeufelberger) [reply RELEASENOTES.md]: isses --> issues

**Pull request merged**